docs/data-ai/ai/train-tflite.md

```diff
@@ -26,53 +26,45 @@ date: "2024-12-03"
 ---
 
 Many machines have cameras through which they can monitor their environment.
-With machine leaning, you can train models on patterns within that visual data.
-You can collect data from the camera stream and label any patterns within the images.
-
-If a camera is pointed at a food display, for example, you can label the image of the display with `full` or `empty`, or label items such as individual `pizza_slice`s.
-
-Using a model trained on such images, machines can make inferences about their environments.
-Your machines can then automatically trigger alerts or perform other actions.
-If a food display is empty, the machine could, for example, alert a supervisor to restock the display.
-
-Common use cases for this are **security**, **quality assurance**, and **food service** applications.
-
+With machine leaning (ML), you can train models on patterns within image data.
 Follow this guide to use your image data to train an ML model, so that your machine can make inferences about its environment.
 
 ## Prerequisites
 
-{{% expand "A running machine connected to the Viam app. Click to see instructions." %}}
+{{% expand "a machine connected to the Viam app" %}}
 
 {{% snippet "setup.md" %}}
 
-{{% /expand%}}
+{{% /expand %}}
 
-{{% expand "A dataset with labels. Click to see instructions." %}}
+{{% expand "a dataset with labels" %}}
 
-Follow the guide to [create a dataset](/data-ai/ai/create-dataset/) if you haven't already.
+Follow the guide to [create a dataset](/data-ai/ai/create-dataset/).
 
 {{% /expand%}}
 
-## Train a machine learning (ML) model
+## Train a machine learning model
 
-Now that you have a dataset with your labeled images, you are ready to train a machine learning model.
+Now that you have a dataset that contains your labeled images, you are ready to train a machine learning model.
 
 {{< table >}}
 {{% tablestep number=1 %}}
 **Train an ML model**
 
```

Comment:
```suggestion
```

Reply:
D'oh. All of these tablesteps should have titles! I added one for the first step, so I think I should leave this one.